### PR TITLE
Ltd 5054 expose elasticsearch score in denial search results

### PR DESCRIPTION
**Pull request by markj0hnst0n**
### Aim

Expose 'score' in the denial results table and add feature flag to turn it on and off

Corresponding frontend PR [here](https://github.com/uktrade/lite-frontend/pull/1992)

[LTD-5054](https://uktrade.atlassian.net/browse/LTD-5054)

[LTD-5054]: https://uktrade.atlassian.net/browse/LTD-5054?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by depsiatwal — COMMENTED** [review]
no tests ?

**Comment by markj0hnst0n** [comment]
I thought it was more relevant to test on the front end as that's where the feature flags and search score reside? @depsiatwal

**Comment by depsiatwal** [comment]
> I thought it was more relevant to test on the front end as that's where the feature flags and search score reside? @depsiatwal

regardless o f how or where code is used . In an ideal world we should have unit tests

**Comment by markj0hnst0n** [comment]
I've added one in there 🙂

**Review by depsiatwal — COMMENTED** [review]
LGTM
is there a UI PR this can be linked to ?

**Comment by markj0hnst0n** [comment]
@depsiatwal 
> LGTM is there a UI PR this can be linked to ?

https://github.com/uktrade/lite-frontend/pull/1992

File: api/external_data/tests/test_views.py [hunk 1]
```diff
@@ -420,6 +420,30 @@ def test_denial_entity_search(self, query, expected_items):
         regime_reg_ref_results = [r["regime_reg_ref"] for r in response_json["results"]]
         self.assertEqual(regime_reg_ref_results, expected_items)
 
+    @pytest.mark.elasticsearch
+    @parameterized.expand(
+        [
+            ({"search": "name:(Organisation Name)"}, [1.19, 1.01, 1.01]),
+            ({"search": "name:(The Widget Company)"}, [3.91]),
+            ({"search": "name:(XYZ)"}, [1.3]),
+            ({"search": "address:(Street Name)"}, [0.58, 0.53, 0.53]),
```
[the remark this answers is not in the document]
Comment (depsiatwal) [reply api/external_data/tests/test_views.py:429]: yeah I was thinking kind of similar is this introducing a flaky test score can't be replied on to be the same

[the remark this answers is not in the document]
Comment (markj0hnst0n) [reply api/external_data/tests/test_views.py]: I've refactored this test

**Review by depsiatwal — APPROVED** [review]
lgtm

**Pull request merged**